src/js/utils/dom.js

```diff
@@ -81,6 +81,7 @@ function onWindowBlur() {
 }
 
 export function monitorBrowserState() {
+    store.dispatch(hasFocus(document.hasFocus()));
```

Comment:
compatibility seems alright for `document.hasFocus` on the main desktop browsers, but it's not clear on mobile (and opera doesn't support it). I would just do `document.hasFocus ? document.hasFocus() : true` or wrap it in a if and change the default reducer value to true.